src/survey/factory.ts

```diff
@@ -0,0 +1,81 @@
+import { Factory } from 'src/common'
+import { Survey } from './survey'
+import { Labels, LanguageCode } from 'src/language'
+
+export type SurveyFactoryParams = {
```

Comment:
better to call this SurveyCreateOptions ?

Reply:
maybe SurveyCreateParams, not options